async_firebase/client.py

```diff
@@ -162,19 +162,19 @@ def build_android_config(  # pylint: disable=too-many-locals
         priority: str,
         ttl: t.Union[int, timedelta] = DEFAULT_TTL,
         collapse_key: t.Optional[str] = None,
-        restricted_package_name: str = None,
-        data: t.Dict[str, t.Any] = None,
-        title: str = None,
-        body: str = None,
-        icon: str = None,
-        color: str = None,
-        sound: str = None,
-        tag: str = None,
-        click_action: str = None,
-        body_loc_key: str = None,
-        body_loc_args: t.List[str] = None,
-        title_loc_key: str = None,
-        title_loc_args: t.List[str] = None,
+        restricted_package_name: t.Optional[str] = None,
```

Comment:
Just a suggestion and nothing that prevents this PR from being merged - I would rather update type annotations in a separate PR

Reply:
good call out, I promise to keep that in mind with the next set of changes like this ones. ))